Doc/library/functions.rst

```diff
@@ -81,9 +81,26 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: bin(x)
 
-   Convert an integer number to a binary string. The result is a valid Python
-   expression.  If *x* is not a Python :class:`int` object, it has to define an
-   :meth:`__index__` method that returns an integer.
+   Convert an integer number to a binary string prefixed with "0b". The result is
+   a valid Python expression. If *x* is not a Python :class:`int` object, it has
+   to define an :meth:`__index__` method that returns an integer.
+
+   Some examples:
+
+      >>> bin(12)
+      '0b1100'
+      >>> bin(-10)
+      '-0b1010'
+
+   If prefix "0b" is desired or not, you can use either of the following ways.
+
+      >>> format(14, '#b'), format(14, 'b')
+      ('0b1110', '1110')
+      >>> f'{14:#b}', f'{14:b}'
+      ('0b1110', '1110')
+
+   See also :
```

Comment:
I don't understand why :func:`format` is splitted in two lines. I expect an error when building the doc. Please don't split this statement into two lines.